superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeControl.test.tsx

```diff
@@ -31,8 +31,8 @@ import { testWithId } from 'src/utils/testUtils';
 import {
   EchartsMixedTimeseriesChartPlugin,
   EchartsTimeseriesChartPlugin,
-} from '@superset-ui/plugin-chart-echarts/lib';
-import { LineChartPlugin } from '@superset-ui/preset-chart-xy/lib';
```

Comment:
I can't remember why we imported from `/lib` here, but if this works, hooray!

Reply:
I think `import from /lib` was from IDE auto-generate code. `import from /lib` depend on package build, so we should avoid this pattern.